modules/rest-api/pages/rest-set-up-services-existing-nodes.adoc

```diff
@@ -0,0 +1,140 @@
+----
+
+These are services related parameters:
+
+* `knownNodes`: The value is a list containing all nodes that are in the cluster at the start of the rebalance process.
+
+* `ejectedNodes`: The value is a list containing all nodes that are to be ejected from the cluster during the rebalance process.
+
+* `topology`: The value is a list of services and the nodes in which services must reside, after the rebalance operation completes.
++
+The value is `topology[<service>]=<list of nodes>`.
++
+Where,
+
+** `<service>` can be one of the following services:
+
+*** `fts` for the Search Service.
+*** `index` for the Index Service.
+*** `n1ql` for the Query Service.
+*** `backup` for the Backup Service.
+*** `cbas` for the Analytics Service.
+
+** `<list of nodes>` indicate the nodes on which the `<service>` must reside in after the rebalance operation is complete.
+The node names are separated by commas.
++
+If a `<service>` is running on any node in the cluster that's not in this list, Couchbase Server removes that `<service>` during the rebalance.
+See xref:rest-api/rest-cluster-rebalance.adoc[Rebalancing the Cluster].
+
+** For example, `-d topology[<service>]=otp_node1,otp_node2`.
+
+NOTE: The rebalance occurs even if no `<service>` is changed.
+
+Each assigned service is allocated a default amount of memory.
+You can modify these allocations as needed.
+
+For information about allocating memory per service, see xref:rest-api:rest-configure-memory.adoc[Configuring Memory].
+
+To fetch the details of current memory allocation for each service, see xref:rest-api:rest-get-memory-information.adoc[Getting Memory Information].
+
+== Required Permissions
+
+`Full Admin` role.
+
+[#responses]
+== Responses
+
+|===
+|Value | Description  
+
+| `200 OK` and JSON array containing the expired backups and their details.
+| Successful call.
+
+| `400`
+| Invalid parameter.
+
+| `400 Object Not found`
+| The repository in the endpoint URI does not exist.
+
+| `401 Unauthorized`
+|  Authorization failure due to incorrect username or password.
+
+| `403 Forbidden`, plus a JSON message explaining the minimum permissions.
+| The provided username has insufficient privileges to call this method.
+
+| `404 Object Not Found`
+| Error in the URI path.
+
+| `500 Internal Server Error`
+| Error in Couchbase Server.
+|===
+
+=== Example
+
+For illustration, consider 5 nodes in the cluster with the following services:
+
+* 172.23.108.69 – Data, Index, Query
+* 172.23.108.70 – Data, Search, Query
+* 172.23.108.71 – Query
+* 172.23.108.72 – Backup
+* 172.23.108.73 – Analytics
+
+Reconfigure the services on the existing nodes so that the following is the required output of services on each of the 5 nodes:
+
+* 172.23.108.69 – Data, Query
+* 172.23.108.70 – Data, Index
+* 172.23.108.71 – Index, Query
+* 172.23.108.72 – Search, Query
+* 172.23.108.73 – Analytics
+
+Run the following command to reconfigure the services on the existing nodes and get the required output.
+
```

Comment:
Note that since the Backup service is not in the topology list, the Backup service will be removed from the cluster.  Also, there are 2 nodes that did not previously have any services using the disk storage path for Indexes that will have services using the Indexes disk storage path after the cluster topology change has completed with the rebalance.